### ISSUE TITLE
Exporting Newest Version

### ISSUE THREAD

**Issue by Norbiros** [jensjeflensje/websk#25]
Hello,
I wanted do add something to code. I coppied it to my repository. And i exported it. But when u lokk in code, there are some mistakes.

First of all, on what version i must export this plugin? Java 8 or Java 9

In `pom.xml` u can find:
`<java.version>1.8</java.version>`
`<source>9</source><target>9</target>`
So i don't know if i can use 8 or 9.

When i try this on 8 i have this in console:
```
[ERROR] [ERROR] Some problems were encountered while processing the POMs:
[ERROR] Malformed POM /home/norbi/.local/share/Trash/files/websk/pom.xml: Unrecognised tag: 'goal' (position: START_TAG seen ...</phase>\n                        <goal>... @38:31)  @ /home/norbi/.local/share/Trash/files/websk/pom.xml, line 38, column 31
[WARNING] 'dependencies.dependency.version' for org.jetbrains:annotations:jar is either LATEST or RELEASE (both of them are being deprecated) @ line 104, column 22
 @ 
[ERROR] The build could not read 1 project -> [Help 1]
[ERROR]   
[ERROR]   The project dev.jensderuiter:websk:1.2.0 (/home/norbi/files/websk/pom.xml) has 1 error
[ERROR]     Malformed POM /home/norbi/files/websk/pom.xml: Unrecognised tag: 'goal' (position: START_TAG seen ...</phase>\n                        <goal>... @38:31)  @ /home/norbi/files/websk/pom.xml, line 38, column 31 -> [Help 2]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/ProjectBuildingException
[ERROR] [Help 2] http://cwiki.apache.org/confluence/display/MAVEN/ModelParseException
```

When i try exporting data from `/release` it works fine.
Newest version 1.2.0 was published by @SkyCraft78. I think he exported this version on his computer, but he don't pull all changes. Can someone look at it?

After testing i found 1.2.0 need Java version 9. I will update that on my computer, change 1.8 to 1.9 and test it again.

**Comment by Norbiros** [comment]
I think problem is with this:
```
<goals>
     <goal>shade</goal>
</goals>```

How it must look?

**Comment by Norbiros** [comment]
After exporting using `mvn clean install` u can get error:
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.8.1:compile (default-compile) on project websk: Fatal error compiling: invalid target release: 9 -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
```

**Comment by Norbiros** [comment]
Everythng is fixed in newest version 1.2.1, thanks!
